control-plane/pkg/config/env_config.go

```diff
@@ -44,6 +44,7 @@ type Env struct {
 	SystemNamespace         string `required:"true" split_words:"true"`
 	ContractConfigMapFormat string `required:"true" split_words:"true"`
 	DefaultBackoffDelayMs   uint64 `required:"false" split_words:"true"`
+	IngressPodTlsPort       string `required:"false" split_words:"true"`
```

Comment:
@Rahul-Kumar-prog the GitHub actions are failing with the message:

> other declaration of IngressPodTlsPort

You can find that definition here: https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/control-plane/pkg/config/env_config.go#L44

Reply:
@Rahul-Kumar-prog it looks like this is happening because your branch is behind the main branch by 23 commits (one of which adds `IngressPodTlsPort`). 

The tests generally try to run your changes on top of whatever code is currently on the main branch, which is leading to this error.

To fix this, please merge this repo's `main` back into your branch